src/app/page.tsx

```diff
@@ -1,9 +1,19 @@
+import AboutEventTile from "@/components/LandingPage/AboutEventTile";
 import JudgingCriteria from "@/components/LandingPage/JudgingCriteria";
 import PagePlaceholder from "@/components/PagePlaceholder";
 import { enableLandingPage } from "@/featureFlags";
 
 export default function Home() {
   return (
-    <main>{enableLandingPage ? <PagePlaceholder /> : <JudgingCriteria />}</main>
+    <main>
+      {enableLandingPage ? (
```

Comment:
ok this isn't in your PR but this is a bit confusing because `enableLandingPage` is defaulted to true and it enables the placeholder (doesn't seem intuitive)

could you change `enableLandingPage` to default to false